buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfigImpl.kt

```diff
@@ -45,4 +48,12 @@ data class FladleConfigImpl(
   override val outputStyle: Property<String>,
   override val legacyJunitResult: Property<Boolean>,
   override val fullJunitResult: Property<Boolean>
-) : FladleConfig
+) : FladleConfig {
+  fun sanityRoboRun() {
```

Comment:
I like this method. Its a good compromise!
Can we rename it to `clearPropertiesForSanityRobo()`? And add a bit of javadoc explaining which fields are being cleared?